source/appModules/foobar2000.py

```diff
@@ -29,28 +45,47 @@ def event_gainFocus(self, obj, nextHandler):
 
 	def getElapsedAndTotal(self):
 		if not self.statusBar: return None
-		text = self.statusBar.firstChild.name
+		statusBarContents = self.statusBar.firstChild.name
 		try:
-			ltime = text.split("|")[4].split(" / ")
-		except IndexError:
-			return None
-		elapsedTime = calendar.timegm(time.strptime(ltime[0].strip(),getFormat(ltime[0])))
-		totalTime = calendar.timegm(time.strptime(ltime[1].strip(),getFormat(ltime[1])))
-		return elapsedTime,totalTime
+			playingTimes = statusBarContents.split("|")[4].split("/")
+			return playingTimes[0], playingTimes[1]
+		except:

```

Comment:
I'm a little wary of having a catch-all exception block, which just swallows the exception. Perhaps catch the specific ones you know about and log others?